nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -66,14 +66,8 @@
 import javax.net.ssl.TrustManager;
 import javax.net.ssl.TrustManagerFactory;
 import javax.net.ssl.X509TrustManager;
-import okhttp3.Cache;
-import okhttp3.Credentials;
-import okhttp3.MediaType;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.RequestBody;
-import okhttp3.Response;
-import okhttp3.ResponseBody;
+
+import okhttp3.*;
```

Comment:
Our checkstyle prohibits wildcard imports. You can download the checkstyle definitions for your IDE from the [Contributor Guide](https://cwiki.apache.org/confluence/display/NIFI/Contributor+Guide) and run the Maven build locally with `-Pcontrib-check` to see and resolve any issues.

Reply:
First of all thanks for your review and sorry for my english spelling. you are right, It was auto import with shortcut. I fix it. I download checkstyle.